supersaintex/saint-sorting
# Make shell script to run the setup and test of web application

## Description
I made the shell script to run the setup and test of the web application.

## Test-Performed
1. run selenium
2. run the new script in /saint-sorting/

You'll confirm you can run setup and test at one window.
